lib/matplotlib/lines.py

```diff
@@ -1257,13 +1255,7 @@ def update_from(self, other):
         self._drawstyle = other._drawstyle
 
     def _get_rgba_face(self, alt=False):
-        facecolor = self._get_markerfacecolor(alt=alt)
-        if (isinstance(facecolor, six.string_types)
-                and facecolor.lower() == 'none'):
-            rgbaFace = None
-        else:
-            rgbaFace = mcolors.to_rgba(facecolor, self._alpha)
-        return rgbaFace
+        return mcolors.to_rgba(self._get_markerfacecolor(alt=alt), self._alpha)
```

Comment:
The same as above

Reply:
to_rgba already handles the lowercasing (https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/colors.py#L146).